wa/starter/src/main/resources/application.properties

```diff
@@ -40,9 +40,14 @@ management.endpoint.loggers.enabled=true
 management.endpoint.refresh.enabled=true
 management.endpoint.registeredServices.enabled=true
 
-cas.events.core.enabled=false
+# Cache service definitions for 5 minutes
+cas.service-registry.cache.duration=PT5M
+
+# Reload services and hydrate the cache every 5 minutes
+cas.service-registry.schedule.repeat-interval=PT5M
+cas.service-registry.schedule.start-delay=PT30S
 
-server.servlet.session.tracking-modes=cookie
```

Comment:
Any reason to remove `server.servlet.session.tracking-modes=cookie` ?

Reply:
It appears that this line is unnecessary and a duplicate in fact. Here's the first reference:
https://github.com/apache/syncope/blob/master/wa/starter/src/main/resources/application.properties#L31

Reply:
Nice catch :-)